core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -327,6 +327,24 @@ public T keepAliveWithoutCalls(boolean enable) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Sets the retry buffer size in bytes. The implementation may only estimate the buffer size being
+   * used instead of counting the exact memory allocated. It does not have any effect if retry is
+   * disabled.
+   */
+  public T retryBufferSize(long bytes) {
```

Comment:
The global one, `retry_buffer_size_in_bytes` in the spec, according to @markdroth is per channel

Reply:
Okay. So it isn't global. That's fine. Can someone take the action item of specifying that those two options are per-channel?

Reply:
@ncteisen should weigh in on this.  @dfawley may be interested as well.

FWIW, in C-core, I have implemented a per-call limit, but I have not (yet?) implemented a global or per-channel limit.  We'll need to discuss what makes the most sense for the latter.

I had originally not paid that much attention to what the spec said about the type of buffer limitations that can be imposed, since I was thinking that it would be okay to let each implementation make their own decisions about this; the thing that I thought was more important was the spec about how the client should react when a buffer limit is hit.  However, we've been getting more feedback that it would be useful for our implementations to provide consistent behavior for things like this, so we may want to look at the details of this more closely.

Reply:
I think that having `retry_buffer_size_in_bytes` be global is fine. Unless there is a common need for per-channel granularity, the simpler option makes more sense.

Reply:
I suspect it's probably easier to have a per-channel limit.  Otherwise, we need to introduce a global lock of some sort, which we don't otherwise need.

For example, in C-core, this limit would probably be introduced via the ResourceQuota mechanism, which is configured on a per-channel basis.